app/controllers/policy_classes_controller.rb

```diff
@@ -89,10 +96,10 @@ def ensure_can_assess_planning_application
   end
 
   def status
-    if params[:commit].downcase.match(/mark as complete/).present?
-      :complete
-    else
-      :in_assessment
-    end
+    commit_matches?(/mark as complete/) ? :complete : :in_assessment
+  end
+
+  def commit_matches?(regex)
+    params[:commit].downcase.match(regex).present?
```

Comment:
I don't relying on the button text like this, but I don't know of another way to change what we do based on which of several buttons was used to submit the same form. Any ideas?